Makefile

```diff
@@ -1,143 +1,11 @@
-xml2rfc ?= "xml2rfc"
-saxpath ?= "lib/saxon9.jar"
-saxon ?= java -classpath $(saxpath) net.sf.saxon.Transform -l
-kramdown2629 ?= XML_RESOURCE_ORG_PREFIX=https://xml2rfc.tools.ietf.org/public/rfc kramdown-rfc2629
+LIBDIR := lib
```

Comment:
Add USE_XSLT := true here.

Reply:
Yes, that's the extra commit on the other branch.  :-)